### PR TITLE
Docs: Enterprise 6.6

### PR DESCRIPTION
**Pull request by xlson**
**What this PR does / why we need it**:

Covers:
- White-labeling
- Setting up repositories for enterprise

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by xlson** [comment]
@oddlittlebird I'm in the installation instructions poking again, trying to get enterprise instructions in there. It's not great but not sure if we can do it much better right now without new ways to allow the user to make choices to limit what kind of instruction we show.

File: docs/sources/enterprise/white-labeling.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+{{< docs-imagebox img="/img/docs/v66/whitelabeling_1.png" max-width="800px" caption="White labeling example" >}}
+
+This release adds new white labeling options to the `grafana.ini` file (can also be set via ENV variables).
+
+You can change the following elements:
+
+- Login Background
+- Logos and Favorite icon
+- Footer and sidebar links
```
Comment (torkelo) [inline docs/sources/enterprise/white-labeling.md]: Better to be explicit here
- Login Background
- Login Logo
- Side menu top logo
- Footer & Help menu links
 - Fav icon (shown in browser tab)

File: docs/sources/installation/debian.md [hunk 2]
```diff
@@ -26,26 +26,40 @@ If you install from the APT repository, then Grafana is automatically updated ev
 
 The APT repository installs the open source edition of Grafana.
```
Comment (torkelo) [inline docs/sources/installation/debian.md]: This line is no longer correct / confusing given the table below

**Review by torkelo — COMMENTED** [review]
Need to add the white labeling page to the enterprise menu in docs/sources/menu.yaml

File: docs/sources/installation/debian.md [hunk 3]
```diff
@@ -26,26 +26,40 @@ If you install from the APT repository, then Grafana is automatically updated ev
...
+
+# Alternatively you can add the beta repository, see in the table above
 sudo add-apt-repository "deb https://packages.grafana.com/oss/deb stable main"
+
 sudo apt-get update
 sudo apt-get install grafana
 ```
 
-#### To install the latest beta release:
+#### To install the latest stable enterprise release:
```
Comment (oddlittlebird) [inline docs/sources/installation/debian.md]: Capitalize "Enterprise" throughout.

File: docs/sources/installation/rpm.md [hunk 4]
```diff
@@ -48,15 +48,25 @@ sudo yum install <rpm package url>
...
 1. Add a new file to your YUM repo using the method of your choice. The command below uses `nano`.
+
 ```bash
 sudo nano /etc/yum.repos.d/grafana.repo
 ```
+
 2. Enter the following information in grafana.repo and then save it.
 
-For stable releases:
+> For beta, use the url in the table above
```
Comment (oddlittlebird) [inline docs/sources/installation/rpm.md]:
```suggestion
> For beta, use the URL in the table above
```

File: docs/sources/installation/rpm.md [hunk 5]
```diff
@@ -48,15 +48,25 @@ sudo yum install <rpm package url>
...
+
+For oss releases:
```
Comment (oddlittlebird) [inline docs/sources/installation/rpm.md]:
```suggestion
For OSS releases:
```

**Comment by xlson** [comment]
Thanks @oddlittlebird @torkelo !

**Comment by oddlittlebird** [comment]
@papagian - The change in the repositories is not mentioned in the release notes.

@marcusolsson , @RichiH - Do we need to manage the message on this APT and YUM repository change? @daniellee tells me that the OSS community can be very ticklish about that.

**Comment by oddlittlebird** [comment]
Also, why are we removing references to installing beta versions of the software?

**Comment by xlson** [comment]
@oddlittlebird what change in the repositories? They stay the same as they've been, with the difference that enterprise is now documented.

I'm not sure what we should do in regards to documenting the beta vs stable. I choose to make the beta instruction take up less place in the instruction (the url to the repo is still in the table) and focus on stable enterprise and oss instead. Having 4 almost identical sections to cover all of oss/stable, oss/eta, ee/stable and ee/beta felt like it would be too more confusing. And making both enterprise and oss stable the most obvious seemed most important.

That said, I would prefer to make it easy and obvious to install any version.

**Comment by papagian** [comment]
> @papagian - The change in the repositories is not mentioned in the release notes.

@oddlittlebird 
If you referring to Changelog, this will be generated during the release provided that the respective features are labeled as `add to changelog`.
If you mean the `What's new in 6.6` article, white labeling is [there](https://github.com/grafana/grafana/blob/master/docs/sources/guides/whats-new-in-v6-6.md#enterprise-white-labeling). I don't know if we need to add something else, but if we do @xlson please update the `What's new in 6.6` document and I will take care.

File: docs/sources/installation/debian.md [hunk 6]
```diff
@@ -24,26 +24,40 @@ You can install Grafana using our official APT repository, by downloading a `.de
...
+|-----------------|---------|------------|
+| Grafana OSS     | grafana | `https://packages.grafana.com/oss/deb stable main` |
+| Grafana OSS (Beta)     | grafana | `https://packages.grafana.com/oss/deb beta main` |
+| Grafana Enterprise     | grafana-enterprise | `https://packages.grafana.com/enterprise/deb stable main` |
+| Grafana Enterprise (Beta)     | grafana-enterprise | `https://packages.grafana.com/enterprise/deb beta main` |
 
-#### To install the latest stable release:
+> We recommend all users to install the Enterprise Edition of Grafana, which can be seamlessly upgraded with a Grafana Enterprise [subscription](https://grafana.com/products/enterprise/?utm_source=grafana-install-page).
+
+#### To install the latest Enterprise release:
```
Comment (xlson) [inline docs/sources/installation/debian.md]:
```suggestion
#### To install the latest Enterprise Edition:
```

@torkelo what do you prefer here? I'm partial to Enterprise release but I don't really have a strong opinion.

Comment (torkelo) [reply docs/sources/installation/debian.md]: To install the latest Enterprise edition: 

(lowercase e)

**Review by torkelo — APPROVED** [review]
Will need more revision based on feedback, but I think this is good enough to push live

**Comment by xlson** [comment]
@torkelo Agreed. I'll leave it open until @oddlittlebird has taken a final look and merge before the end of the day.

**Comment by oddlittlebird** [comment]
@xlson , @papagian , I think we need to call out in the release notes that the new repositories are available for YUM and APT repository users. This will let existing users know they can change if they want to, perhaps tell them why they might want to. 
| Grafana Version            | Package            | Repository                                         |
|----------------------------|--------------------|----------------------------------------------------|
| Grafana OSS                | grafana            | `https://packages.grafana.com/oss/rpm`             |
| Grafana OSS (Beta)         | grafana            | `https://packages.grafana.com/oss/rpm-beta`        |
| Grafana Enterprise         | grafana-enterprise | `https://packages.grafana.com/enterprise/rpm`      |
| Grafana Enterprise (Beta)  | grafana-enterprise | `https://packages.grafana.com/enterprise/rpm-beta` |

**Pull request merged**